src/style.toit

```diff
@@ -2,6 +2,85 @@
 // Use of this source code is governed by an MIT-style license that can be
 // found in the LICENSE file.
 
+import .common
+import .element as element
+
 ALIGN_LEFT ::= 0
 ALIGN_CENTER ::= 1
 ALIGN_RIGHT ::= 2
+
+class Style:
+  iterate_properties [block] -> none:
+    unreachable  // Unimplemented.
```

Comment:
Unimplemented as "TODO", or "Unimplemented", as "shouldn't be implemented"?
If it's the TODO, then `throw "UNIMPLEMENTED"`, otherwise change the comment.

Reply:
This is fixed in a later PR.